security/AccessControl.rst

```diff
@@ -18,11 +18,22 @@ that a user or group may perform, whether to invoke a function or access
 data, are organized into a hierarchy, and privileges may be granted at
 any level.
 
+Users and Groups can be updated using the Users and Groups Kylo admin pages:
```

Comment:
The Users and Groups pages are only displayed when the auth-kylo Spring profile is enabled. What about rephrasing the text to: "Users and groups can be managed using the Users and Groups admin pages in Kylo. These pages may not be available if the auth-kylo Spring profile is disabled."